library/std/tests/floats/lib.rs

```diff
@@ -3,9 +3,9 @@
 use std::fmt;
 use std::ops::{Add, Div, Mul, Rem, Sub};
 
-/// Verify that floats are within a tolerance of each other, 1.0e-6 by default.
+/// Verify that floats are within a tolerance of each other, 1.0e-4 by default so we do not break test in Miri.
 macro_rules! assert_approx_eq {
-    ($a:expr, $b:expr) => {{ assert_approx_eq!($a, $b, 1.0e-6) }};
+    ($a:expr, $b:expr) => {{ assert_approx_eq!($a, $b, 1.0e-4) }};
```

Comment:
I think I'd rather not change this for *all* tests, and in particular not for `f64`.

An alternative would be to just set this precision with the affected tests in `library/std/tests/floats/f32.rs`.

Reply:
Maybe the precision should be only loosened only with `cfg!(miri)` as well. I think it is beneficial to learn which platforms, if any, have worse precision than we expect and address that on a case-by-case basis.